src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -825,6 +825,16 @@ def aks_update(
     # DO NOT MOVE: get all the original parameters and save them as a dictionary
     raw_parameters = locals()
 
+    # Check if auto_upgrade_channel is set to "none"
```

Comment:
Can we move these into the AKSManagedClusterUpdateDecorator?

Reply:
Done